packages/devtools_app/lib/src/memory/flutter/memory_heatmap.dart

```diff
@@ -89,7 +222,10 @@ class FlameChartRenderObject extends RenderBox {
     markNeedsPaint();
   }
 
-  // TODO(terry): implemented not yet used to be used by search.
+  Node externalFindNode(String searchName) {
```

Comment:
what does the word "external" mean?
Perhaps remove the word external and make methods that should not be used outside this class private.

Reply:
That was a bad word should have used exposeFindNode, exposeSelectNode would be better.